holoviews/core/layout.py

```diff
@@ -16,7 +16,10 @@
 from .dimension import Dimension, Dimensioned, ViewableElement
 from .ndmapping import OrderedDict, NdMapping, UniformNdMapping
 from .tree import AttrTree
-from .util import (unique_array, get_path, make_path_unique, int_to_roman)
+from .util import (
+    unique_array, get_path, make_path_unique, int_to_roman,
+    unique_iterator
```

Comment:
At this point isn't it better to use ``util`` as a qualified namespace?

Reply:
Sure.

Reply:
Definitely makes sense when you have 4+ imports from ``util``.